rebasehelper/application.py

```diff
@@ -576,7 +576,10 @@ def build_packages(self):
                 build_dict['binary_package_build_error'] = six.text_type(e)
                 results_store.set_build_data(version, build_dict)
 
-                if e.logfile is None:
+                if e.exact_error:
+                    # value acquired from koji.getTaskResult by raising koji.BuildError exception
+                    msg = e.exact_error + ' in ' + results_dir
```

Comment:
What exactly does this print? Do you have an example?

Reply:
koji error message + location of where the log is located.
```error building package (arch noarch), mock exited with status 1; see build.log for more information in /home/skisela/fedora/rubygem-bundler/rebase-helper-results/new-build```

Reply:
Maybe there should also be code checking if there is some *.log at all, even though majority of build errors contains it.

Reply:
Right, so it relies on the format of the error message. That is not the best thing to do, as it could change anytime, but since you do it anyway, you could as well parse the mock error code out of it and use it to determine the logfile.